Cartfile.resolved

```diff
@@ -1,2 +1,3 @@
-github "ReactiveX/RxSwift" "3.0.1"
-github "daltoniam/Starscream" "2.0.2"
+github "ReactiveX/RxSwift" "3.3.1"
```

Comment:
I didn't choose it, carthage updated it to the latest version compatible with ~3.0